packages/contracts-bedrock/test/vendor/Initializable.t.sol

```diff
@@ -24,15 +31,17 @@ contract Initializer_Test is Bridge_Initializer {
     /// @notice Contains the address of an `Initializable` contract and the calldata
     ///         used to initialize it.
     struct InitializeableContract {
+        string name;
         address target;
         bytes initCalldata;
-        uint8 initializedSlotVal;
     }
 
-    /// @notice Contains the addresses of the contracts to test as well as the calldata
-    ///         used to initialize them.
+    /// @notice Array of contracts to test.
     InitializeableContract[] contracts;
 
+    /// @notice Mapping of nickname to actual contract name.
+    mapping(string => string) nicknames;
```

Comment:
Maybe this will become clear as I keep reading, but wonderin why we need nicknames and can't just use actual contract names